search/migrations/0012_insert_profile_2020.py

```diff
@@ -0,0 +1,94 @@
+# Generated by Django 3.0.7 on 2020-08-23 10:17
+
+from django.db import migrations
+import csv
+import re
+
+from psycopg2.extras import NumericRange
+
+from search.models import SubjectName, LanguageName
+
+
+def load(apps, schema_editor):
+    School = apps.get_model("search", "School")
+    HighSchoolClass = apps.get_model("search", "HighSchoolClass")
+    ExtendedSubject = apps.get_model("search", "ExtendedSubject")
+    Language = apps.get_model("search", "Language")
+
+    with open("csvs/publiczne_oddzialy_2020.csv", newline="") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=";")
+        for n, row in enumerate(csv_reader):
+            if n > 0:
+                regon = row[7].zfill(9)
+                school = School.objects.filter(public_institution_data__regon=regon)
+                if (
+                    not school.exists()
```

Comment:
Sooo we just ignore new schools? @micorix also wanted to research REGONs more, whether they can change, he also mentioned problems where a REGON is applied to zespół szkół instead of separate ones? Idk the details.

Reply:
I manually edited the csv a bit: removed schools with no REGON (I think all but one were from outside of Warsaw) and filled missing REGONs where they were easily found. In the end, there was only 1 high school („liceum”) that was in the spreadsheet but not in the db, and it was the one of those I manually pasted REGON for.

Reply:
> removed schools with no REGON (I think all but one were from outside of Warsaw) and filled missing REGONs where they were easily found
@annkamsk Sry, just to confirm. Will we have in db all schools from csv?

Reply:
All „liceum”s with regons

Reply:
All classes from them to be precise. I didn’t update any data on schools yet

Reply:
Ok.

Yet..? :) Does it mean I can assign you to #16 ? Also I would unlink that issue coz you mentioned that this PR does not cover some things described in #16. (i think we can't prevent auto closing the issue)

Reply:
Yeah, you’re right, it shouldn’t be linked to this issue. No, don’t assign me